samer/css/style.css

```diff
@@ -0,0 +1,331 @@
+*, body, html{
+    margin: 0;
+    padding: 0;
+    scroll-behavior: smooth;
+
+}
+
+body{
+background-color: white;
+margin: 1px auto;
+width: 90%;
+}
+
+/* -----------------------starting menu nav------------------- */
+.menu{
+    background: rgb(245,246,246); /* Old browsers */
+    background: -moz-linear-gradient(top, rgba(245,246,246,1) 0%, rgba(219,220,226,1) 21%, rgba(184,186,198,1) 49%, rgba(221,223,227,1) 80%, rgba(245,246,246,1) 100%); /* FF3.6-15 */
+    background: -webkit-linear-gradient(top, rgba(245,246,246,1) 0%,rgba(219,220,226,1) 21%,rgba(184,186,198,1) 49%,rgba(221,223,227,1) 80%,rgba(245,246,246,1) 100%); /* Chrome10-25,Safari5.1-6 */
+    background: linear-gradient(to bottom, rgba(245,246,246,1) 0%,rgba(219,220,226,1) 21%,rgba(184,186,198,1) 49%,rgba(221,223,227,1) 80%,rgba(245,246,246,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f5f6f6', endColorstr='#f5f6f6',GradientType=0 );
+    border-radius: 10px;
+    box-shadow: -1px 6px 3px rgba(124, 120, 120, 0.3);
+    position: fixed;
+    width: 90%;
+    z-index: 100;
+
+}
+ul {
+    text-align: center;
+}
+li {
+    
+    display: inline-block;
+    padding: 15px 50px;
+
+}
+.item {
+    text-decoration: none;
+    color: rgb(59, 2, 2);
+    padding: 15px;
+
+}
+.item:hover{
```

Comment:
you need to intend your code a bit better so people can read it.
<pre>
.class . 
     style
     style
     style
</pre>
and NOT
<pre>
.class
    style
style
         style
</pre>